src/test/java/org/opentripplanner/transit/raptor/speed_test/SpeedTest.java

```diff
@@ -168,7 +172,7 @@ private void runTest() throws Exception {
         printProfileStatistics();
 
         // close() sends the results to influxdb
-        registry.close();
+        uploadRegistry.close();
```

Comment:
Last time I tried this I didn't send the mean and average times before the step had elapsed. I will check again.

Reply:
https://github.com/micrometer-metrics/micrometer/blob/main/micrometer-core/src/main/java/io/micrometer/core/instrument/push/PushMeterRegistry.java#L92

Reply:
This seems to always push it when closing the registry